.github/workflows/linting.yml

```diff
@@ -46,3 +46,14 @@ jobs:
       run: mix do deps.get, deps.compile
     - name: Run Credo
       run: mix credo --all
+
+  spelling:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v4
+    - name: Spell Check Repo
+      uses: crate-ci/typos@v1.30.1
+      with:
+        files: lib test/**/*.exs test/**/*.ex
```

Comment:
```suggestion
        files: lib test/**/*.exs test/**/*.ex readme.md
```

Reply:
Unfortunately this task do not support `exclude` flag (it probably should be supported). Added `README.md` as well.